storey/sql_driver.py

```diff
@@ -115,15 +115,30 @@ def _get_where_clause(self, key, sql_table):
             if i != 0:
                 where_clause += " and "
             if sql_table.columns[self._primary_key[i]].type.python_type == str:
-                where_clause += f'{self._primary_key[i]}="{key[i]}"'
+                where_clause += (
+                    rf'{sql_table.name}."{self._primary_key[i]}"="{key[i]}"'
+                    if "mysql" not in sql_table.dialect_options
+                    else rf'{sql_table.name}.{self._primary_key[i]}="{key[i]}"'
+                )
             else:
-                where_clause += f"{self._primary_key[i]}={key[i]}"
+                where_clause += (
+                    rf"""{sql_table.name}."{self._primary_key[i]}"={key[i]}"""
+                    if "mysql" not in sql_table.dialect_options
+                    else rf"{sql_table.name}.{self._primary_key[i]}={key[i]}"
+                )
         return where_clause
 
     def _update_by_key(self, key, data, table):
         where_clause = self._get_where_clause(key, table)
-        update_clause = " ,".join([f'{key}="{value}"' for key, value in data.items() if key not in self._primary_key])
-        sql_statement = f"UPDATE {table} SET {update_clause} where {where_clause}"
+        if "mysql" not in table.dialect_options:
+            update_clause = " ,".join(
+                [rf'"{key}"="{value}"' for key, value in data.items() if key not in self._primary_key]
+            )
+        else:
+            update_clause = " ,".join(
+                [rf'[{key}]="{value}"' for key, value in data.items() if key not in self._primary_key]
+            )
```

Comment:
Too much duplication
```suggestion
        quote = "" if "mysql" in table.dialect_options else ""
        update_clause = " ,".join(
            [f'[{quote}{key}{quote}]="{value}"' for key, value in data.items() if key not in self._primary_key]
        )
```